ftplugin/jsx/splitjoin.vim

```diff
@@ -0,0 +1,23 @@
+if !exists('b:splitjoin_split_callbacks')
```

Comment:
The reason I put the `if !exists` in some places is because, for HTML, and some other filetypes, there are multiple filetypes defined, like "html.handlebars" and such. So there can be some overlap. To be honest, looking at it now, I'm not sure why some ftplugins have that, so maybe I'll go over them and do some cleanup one of these days.

Could you remove the `if !exists` here and check to see how it behaves? If it continues to work for you after doing that, and after removing the html definitions from the javascript ftplugin, it would be better (no need to put html definitions in the js filetype, if they can be plugged in the jsx one).